app/frontend/src/components/forms/submission/MySubmissionsTable.vue

```diff
@@ -172,13 +191,14 @@ export default {
       if (this.submissionList) {
         const tableRows = this.submissionList.map((s) => {
           return {
-            completedDate: this.getStatusDate(s, 'COMPLETED'),
+            // completedDate: this.getStatusDate(s, 'COMPLETED'),
             confirmationId: s.confirmationId,
+            lastEdited: s.draft ? s.createdAt : undefined,
```

Comment:
should this be `lastEdited: s.draft ? s.updatedAt: undefined,` ?

Reply:
Yeah suspect this is why it's not working

Reply:
good catch. as discussed together, using s.createdAt does not allow you to see the updated date when you save your draft after the first time.

Will be taking a look at changing the API response to get a better field for this situation.